docs/solutions/sql_server_always_on_availability_groups/configuration.tf

```diff
@@ -0,0 +1,60 @@
+# configuration.tf - General configuration of SQL Always On deployment
+# Set variables here to meet your needs within your particular environment
+# Check both this file and sql.tf!
+
+# image_id - OCID of the Windows image to use.
+# BE SURE TO UPDATE WITH MOST CURRENT IMAGE.
+variable "image_id" {
+  default = "ocid1.image.oc1.iad.aaaaaaaajlfsi5npxguvhad3v5d5lu7dc3zcylr2csfdexgd6kor3f6zeqeq"
+}
+
+# vcn_dns_name - Set the domain zone, e.g. <vcn_dns_name>.oraclevcn.com .
+variable "vcn_dns_name" {
+  default = "SQLAlwaysOn"
+}
+
+# label_prefix - Set a unique prefix for all resources. Only required if 
+# deploying multiple clusters within the same compartment.
+variable "label_prefix" {
+  default = ""
+}
+
+# vcn_cidr_block - Set IP address pool for the VCN being created for SQL AO.
+variable "vcn_cidr_block" {
+  default = "10.0.0.0/19"
+}
+
+# compartment_name - Set the compartment to use for the configuration.
+# If the compartment does not exist, it will be created.
+# NOTE: Compartments CANNOT be destroyed once created
+variable "compartment_name" {
+  default = "SQLAlwaysOn"
+}
+
+# compartment_description - Set a short description for the compartment. 
+# Only used for new compartments.
+variable "compartment_description" {
+  default = "Compartment created for SQL Always On deployment"
+}
+
+# ad_count - Set number of ADs to use. Minimum 2, Maximum 3.
+variable "ad_count" {
+  default = "3"
```

Comment:
Let's set this to 2 by default.... setting to 3 creates quite a few instances, and users are likely to hit limits.

Reply:
Not blocking, but does lower the barrier to entry for a first run, and makes that first run more likely to succeed.